src/components/NavBar/SearchBar.tsx

```diff
@@ -56,8 +58,10 @@ export const SearchBarDropdownSection = ({
         <Box>{header}</Box>
       </Row>
       <Column gap="12">
-        {suggestions?.map((suggestion, index) =>
-          isCollection(suggestion) ? (
+        {(isLoading ? [...Array(suggestions.length)] : suggestions).map((suggestion, index) =>
```

Comment:
what if we break this thing into a larger ternary operator based around `isLoading`

Reply:
I moved the logic to make the blank array map compatible higher in the call hierarchy to help with readability